subgraphs/uniswap-v3-forks/src/mappings/position-manager.ts

```diff
@@ -157,9 +166,22 @@ export function handleDecreaseLiquidity(event: DecreaseLiquidity): void {
 }
 
 export function handleTransfer(event: Transfer): void {
-  const account = getOrCreateAccount(event.params.to);
   const position = getOrCreatePosition(event, event.params.tokenId);
 
+  const account = getOrCreateAccount(event.params.to);
+  const oldAccount = getOrCreateAccount(position!.account);
```

Comment:
Instead of assuming we have a position, I think we should be checking if it is null and throw a warning (or error) if not. It would crash this, but also mean there is something else wrong that is deeper

Reply:
Should we add a log if the pos is not found? Wouldn't that mean something else is wrong

Reply:
Yeah, good point. I'll add an error here since now it ideally should not occur.